src/api/liskApi.js

```diff
@@ -208,7 +209,7 @@ LiskAPI.prototype.sendRequest = function sendRequest(
 ) {
 	const callback = callbackIfOptions || optionsOrCallback;
 	const options = (typeof optionsOrCallback !== 'function' && typeof optionsOrCallback !== 'undefined')
-		? privateApi.checkOptions.call(this, optionsOrCallback)
+		? utils.checkOptions.call(this, optionsOrCallback)
```

Comment:
All these `utils` functions can be called directly instead of using `.call` or `.bind`.